packages/logging/logging.js

```diff
@@ -49,6 +49,18 @@ var LEVEL_COLORS = {
 
 var META_COLOR = 'blue';
 
+// Default colors cause readability problems on Windows Powershell,
+// switch to bright variants. While still capable of millions of
+// operations per second, the benchmark showed a 25%+ increase in
+// ops per second (on Node 8) by caching "process.platform".
+var isWin32 = process && process.platform === 'win32';
```

Comment:
Unfortunately, just referring to `process` when it isn't defined will throw a `ReferenceError`, so this needs to be
```js
var isWin32 = typeof process === "object" && process.platform === "win32";
```